subiquity/server/controllers/filesystem.py

```diff
@@ -294,6 +294,9 @@ async def _unmount_systems_dir(self):
             self._source_handler = None
 
     async def _get_system(self, variation_name, label):
+        if not self.app.opts.enhanced_secureboot:
```

Comment:
It probably doesn't make any difference in practice, but I think we should continue to offer options that have a snapd_system_label but the model thus referenced does not include a gadget. I think 

```
             if system is not None and len(system.volumes) > 0:
+                if not self.app.opts.enhanced_secureboot:
+                    continue
                 info = self.info_for_system(name, label, system)
```

in _examine_systems would be better.

Reply:
Done